app/helpers/general-date.js

```diff
@@ -1,13 +1,15 @@
 import Helper from '@ember/component/helper';
 import moment from 'moment';
+import { tzAbbr } from 'open-event-frontend/utils/dictionary/tzAbbr';
 
 const dateFormats = {
   'date-time-long'     : 'dddd, D MMMM, YYYY h:mm A',
-  'date-time-tz-long'  : 'dddd, D MMMM, YYYY h:mm A (z)',
+  'date-time-tz-long'  : 'dddd, D MMMM, YYYY h:mm A',
   'date-time-short'    : 'D MMM, YYYY h:mm A',
-  'date-time-tz-short' : 'D MMM, YYYY h:mm A (z)',
+  'date-time-tz-short' : 'D MMM, YYYY h:mm A',
   'date-short'         : 'D MMM, YYYY',
-  'time-tz-short'      : 'h:mm A (z)'
+  'time-tz-short'      : 'h:mm A',
+  'tz'                 : ' '
```

Comment:
Shouldn't it be empty string?

Reply:
No bcz then format will be 'tz' which after passing to moment will convert to for eg tz is Asia/Kolkata then will convert to tIST and then after appending timezoneabbr again will finally give ```tIST (IST)```

So just to get format correct when this line is executed 
```js
let format = (dateFormats[params[1]] || params[1]) || 'h:mm A, MMMM Do YYYY';
```

'tz' : ' '

Reply:
Conclution tz : '' returns false